### ISSUE TITLE
NVDA doesn't function properly in Windows-8 mail app.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3226]
**Reported by Harich on 2013-05-11 16:47**
When tabbing through Windows-8 mail app, NVDA seems to have few troubles. When you reach to view your e-mail, it would not read. You cannot even navigate with up/down arrow keys either.

**Comment by leonardder** [comment]
Not sure about this one. Windows 8 is not exactly abandoned yet, but people have the possibility to update to Windows 10 and effort is being put into the Windows 10 mail app.

**Comment by bhavyashah** [comment]
I agree with @leonardder's https://github.com/nvaccess/nvda/issues/3226#issuecomment-313199261 with respect to the analysis that Microsoft is probably investing efforts in improving the Windows 10 Mail app (and so has NVDA) rather than the Windows 8 Mail app. Since Leonard seems to suggest that the Windows 8 Mail app, at the same time, is not officially declared as abandonware, I suppose this request probably cannot thus be fully overturned in that regard. Thoughts? CC @feerrenrut

**Comment by feerrenrut** [comment]
As already mentioned, the number of users of windows 8 mail is likely small already. NV Access is not able to have a great impact by specifically trying to fix this. However, I will leave the issue open, and mark it as a priority 4. We are happy to provide guidance and accept a PR from someone who wishes to pursue this issue.

**Comment by josephsl** [comment]
Hi, I propose that we either adopt PR’s against Windows 8.1, or close this as wontfix as Windows 8 is an unsupported operating system. Thanks.

**Comment by Brian1Gaff** [comment]
All the people I know on 8 are in fact on 8.1. I'm assuming this version is 
still supported. It would be a bit dangerous to expect everyone has updated 
to 10 simply because the path is there.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by ruifontes** [comment]
Joseph, where is said that NVDA do not support Windows 8.1?

Rui


-----Mensagem Original----- 
De: Joseph Lee
Data: 19 de setembro de 2017 03:13
Para: nvaccess/nvda
Cc: Subscribed
Assunto: Re: [nvaccess/nvda] NVDA doesn't function properly in Windows-8 
mail app. (#3226)

Hi, I propose that we either adopt PR’s against Windows 8.1, or close this 
as wontfix as Windows 8 is an unsupported operating system. Thanks.

**Comment by Adriani90** [comment]
Apps on Windows 8 will not be sustainably developed in the future. So supporting Windows 10 Apps should have higher priority. For Windows 8 I suggest using Thunderbird or MS Outlook for e-mails.

**Comment by Adriani90** [comment]
I am closing this issue as abandoned since Support for Windows 8.1 ended completely in 2020. If you have such issues with the mail app on windows 10 or 11, please open new issues accordingly.
